### PR TITLE
EW-179: Modified blog

### PR DESCRIPTION
**Pull request by Girbons**
### What it does

- Over the post is now visible the user gravatar
- Below each post are now available a maximum of 3 similar posts that match the post tags

### CODE REVIEW

File: django-website/blog/models.py [hunk 1]
```diff
@@ -117,3 +117,16 @@ class Post(Page):
...
+        tags = self.tags.names()
+        articles = Post.objects.live().exclude(pk=self.pk)
+        articles = articles.order_by('-date')
+        context = super().get_context(request)
+        similar_articles = []
+        for article in articles:
+            for tag in tags:
+                if tag in article.tags.names():
+                    similar_articles.append(article)
+        context['similar_articles'] = similar_articles[:3]
```
Comment (silvanocerza) [inline django-website/blog/models.py]: We don't really need to loop every article if we only need the last 3.

**Review by silvanocerza — CHANGES_REQUESTED** [review]
Check comments

File: django-website/blog/models.py (hunk 1 above)
Comment (Girbons) [reply django-website/blog/models.py]: Are you sure? Of course there is a better way to do this

Comment (silvanocerza) [reply django-website/blog/models.py]: Oh right, you need to check the tags for each one. Nevermind.

**Review by silvanocerza — APPROVED** [review]

**Review by Psykopear — APPROVED** [review]

**Pull request merged**